app/helpers/quadicon_helper.rb

```diff
@@ -191,12 +193,7 @@ def img_for_host_vendor(item)
   end
 
   def img_for_auth_status(item)
-    case item.authentication_status
-    when "Invalid" then "100/x.png"
-    when "Valid"   then "100/checkmark.png"
-    when "None"    then "100/unknown.png"
-    else                "100/exclamationpoint.png"
-    end
+    status_img(item)
```

Comment:
why does this belong to refactoring of the quadicons?

Reply:
Well we need to return url for item's status, and it is better to move these to seperate file which is imported and is easier to read.

Reply:
@karelhala : do we really need the URL?

I mean we figured when working on the GTLs that the URLs are calculated on 2 places and that really matters is the single URL that is actually not part of the quadicon.

What about removing all the URL logic here? Instead of refactoring that part. That might actually save some time and code.

Reply:
@martinpovolny this does not touches urls, these are just paths to images.

Reply:
wanted to say the same :wink: